README.rst

```diff
@@ -17,7 +17,7 @@
 .. |Coverage| image:: https://api.codacy.com/project/badge/Coverage/b92ae35b691141ceb5f2ee74beaf39d3
    :target: https://www.codacy.com/manual/theislab/anndata
 
-anndata - Annotated Data
+anndata - Annotated data
```

Comment:
See: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings